.travis.yml

```diff
@@ -1,5 +1,4 @@
 language: go
-sudo: required
```

Comment:
sudo is required for run minikube - e2e

Reply:
interesting, in Travis Linux build, the `sudo` permission is not quite required.

https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration